src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1644,5 +1654,10 @@ public bool IsOperandOfIncrementOrDecrementExpression(SyntaxNode node)
         {
             return IsOperandOfIncrementExpression(node) || IsOperandOfDecrementExpression(node);
         }
+
+        public string GetLiteralExpressionValueText(SyntaxNode node)
```

Comment:
This one seems unneessary.  you could always just call node.GetFirstToken().ValueText.